lib/fog/image/openstack/v2/models/image.rb

```diff
@@ -162,6 +171,42 @@ def remove_tag(tag)
             requires :id
             service.remove_tag_from_image(id, tag)
           end
+
+          private
+
+          def build_update_json_patch(client_attributes, server_attributes)
```

Comment:
@Ladas as this PR almost 2 months old at this point, I really don't want to touch it again :) While removing duplicate code is nice, I don't think this blocks the merging of this PR.